client/src/i18n/fr/report.json

```diff
@@ -32,19 +34,24 @@
     "INCOME_EXPENSE":"Rapport  des recettes et des dépenses",
     "INCOME_REPORT":"Rapport des recettes",
     "MONTHLY_BALANCE":"Balance mensuelle",
-    "OPEN_DEBTORS":{
-      "DESCRIPTION":"Ce rapport affiche les débiteurs endettés",
-      "TITLE":"Débiteurs endettés", 
-      "TREE":"Dettes des Débiteurs"
+    "OPEN_DEBTORS": {
+      "TITLE":"Débiteurs endettés",
+      "TREE":"Dettes des Débiteurs",
+      "DESCRIPTION": "Ce rapport montre les débiteurs qui ont une dette avec l'institution et donne un certain contrôle sur l'ordre de tri. Pour une analyse avancée, le rapport peut ajouter la dernière date de facturation et la dernière date de paiement, ainsi que limiter la portée de la recherche par une date.",
+      "SHOW_DATE_LIMIT" : "Limité par Date",
+      "DATE_LIMIT_HELP" : "La limite de la date ne fera que numériser les enregistrements de cette date au présent.",
+      "SHOW_DETAILED_VIEW" : "Inclure les colonnes pour les dernières dates de facturation et de paiement",
+      "SHOW_DETAILED_VIEW_HELP" : "L'activation de cette option comprendra la dernière date à laquelle le débiteur a payé et la dernière date où ils ont été facturés. Ceci est utile pour comprendre quand ils étaient à l'établissement. Cependant, cela rendra la requête plus lent.",
+      "SHOW_UNVERIFIED_TRANSACTIONS" : "Inclure les transactions non vérifiées",
+      "SHOW_UNVERIFIED_TRANSACTIONS_HELP" : "L'activation de cette option comprend les enregistrements du Journal de Saisie qui n'ont pas été vérifiés par le comptable ainsi que les enregistrements du Grand Livre que le comptable a vérifié. Cette vue doit être traitée comme une projection jusqu'à ce que le comptable ait approuvé les transactions du journal et les a posté vers le Grand Livre."
```

Comment:
Should read : L'activation de cette option inclut les données du Journal (données non verifiées) et celles du Grand Livre (données vérifiées). Ce rapport doit être consideré comme une simple projection jusqu'à ce que les données du journal soient postées.